hawkular-alerts-metrics/src/main/java/org/hawkular/alerts/external/metrics/CassCluster.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.external.metrics;
+
+import org.jboss.logging.Logger;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ProtocolVersion;
+import com.datastax.driver.core.Session;
+
+/**
+ * Cassandra cluster representation and session factory.
+ *
+ * @author Lucas Ponce
+ */
+public class CassCluster {
```

Comment:
This class here triggered an alarm.
Do you need to access directly Cassandra ? from hawkular-alerts-metrics ?
I haven't finished the full review and perhaps I can find it.